src/main/java/data/SheetsServiceUtil.java

```diff
@@ -32,10 +32,7 @@
 public class SheetsServiceUtil {
 
   public static Sheets getSheetsService() throws IOException, GeneralSecurityException {
-    GoogleCredential credential = SheetsAuthUtil.authorize();
-
-    return new Sheets.Builder(new UrlFetchTransport(), new JacksonFactory(), credential)
-      .setApplicationName("App Engine default service account")
-      .build();
+    // Create a spreadsheet by using the OAuth tokens.
```

Comment:
Is this comment meant to be a TODO?

Reply:
Yes, I plan on using this class in the future.

Reply:
TODO comments should be structured like:
`// TODO(cxfloeder): Create a spreadsheet by using the OAuth tokens.`